python/zbarmodule.c

```diff
@@ -146,10 +146,10 @@ version (PyObject *self,
     if(!PyArg_ParseTuple(args, ""))
         return(NULL);
 
-    unsigned int major, minor;
-    zbar_version(&major, &minor, NULL);
+    unsigned int major, minor, patch;
+    zbar_version(&major, &minor, &patch);
 
-    return(Py_BuildValue("II", major, minor));
+    return(Py_BuildValue("III", major, minor, patch));
```

Comment:
Is this backward compatible, or will it break python the API/ABI?

Reply:
it is an old way to return python tuple with three elements